### PR TITLE
add some ICEs

### PR DESCRIPTION
**Pull request by henryboisdequin**

### CODE REVIEW

File: ices/80664.rs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+mod inner {
+    pub struct Public;
+}
+
+pub use inner::Public as Reexported;
+
+fn main() {}
```
Comment (Alexendoo) [inline ices/80664.rs:7]: This one needs to be passed into rustdoc with `--output-format json` in order to ICE, you use a `.sh` file to do this, e.g. https://github.com/rust-lang/glacier/blob/master/ices/52334.sh

**Review by Alexendoo — CHANGES_REQUESTED** [review]
Thanks! Of the 2 CI errors one is the rustdoc thing, the other is fixed in master now and will go away if you rebase

File: ices/80664.rs (hunk 1 above)
Comment (henryboisdequin) [reply ices/80664.rs:7]: Done

**Comment by henryboisdequin** [comment]
@Alexendoo Do I need to pass `-Z`?

**Comment by henryboisdequin** [comment]
@Alexendoo How do I fix this error? Thanks

**Comment by Alexendoo** [comment]
It should be passed to `rustdoc` rather than `rustc` that's there currently. Looks like the `-` for stdin trick doesn't work with rustdoc however, so you could have it be

```sh
#!/bin/sh

cat > out.rs << EOF
mod inner {
    pub struct Public;
}
pub use inner::Public as Reexported;
EOF

rustdoc out.rs --output-format json
```

**Review by Alexendoo — COMMENTED** [review]
Glacier didn't detect it ICEing there because rustdoc returns error code 1 unlike rustc, I added d0f08abde401417b1809adcc990e12174b95f932 to fix that

So hopefully one last rebase and it's good to go

**Comment by henryboisdequin** [comment]
@Alexendoo Done ✅

**Review by Alexendoo — APPROVED** [review]
Great, thanks

**Pull request merged**